modules/gsqlite3backend/4.3.1_to_4.7.0_schema.sqlite3.sql

```diff
@@ -10,9 +10,9 @@ BEGIN TRANSACTION;
     last_check            INTEGER DEFAULT NULL,
     type                  VARCHAR(8) NOT NULL,
     notified_serial       INTEGER DEFAULT NULL,
+    account               VARCHAR(40) DEFAULT NULL,
     options               VARCHAR(65535) DEFAULT NULL,
-    catalog               VARCHAR(255) DEFAULT NULL,
-    account               VARCHAR(40) DEFAULT NULL
+    catalog               VARCHAR(255) DEFAULT NULL
```

Comment:
is the temporary table now necessary at all?

Reply:
no!